### PR TITLE
add pages build scan documentation

### LINKED ISSUE #4408: Draft documentation for build tasks and individual scans available

### PR DESCRIPTION
**Pull request by drewbo**
## Changes proposed in this pull request:
- add pages build scan documentation
- close https://github.com/cloud-gov/pages-core/issues/4408

## Security Considerations
None

### CODE REVIEW

File: _pages/pages/documentation/build-scans.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+permalink: /pages/documentation/build-scans/
+layout: docs
+navigation: pages
+sidenav: pages-documentation
+---
+
+
+Builds Scans are a new experimental feature being offered by Pages.
+
+For each site build, users can see scan results which check their website for security vulnerabilities, accessibility violations, and other relevant information to help improve their site. These scans begin running after a build completes. When the scan finishes, the results are available as downloadable HTML files for easy sharing among team members. There are currently two scans available: ZAP Security Scan and Accessibility Scan. If you are interested in being part of the beta test of this new feature, please email us at pages-support@cloud.gov.
```
Comment (apburnes) [inline _pages/pages/documentation/build-scans.md]: We aren't going to scan each site build. Maybe something like `For production site builds...` or `Select site builds...`

**Review by apburnes — COMMENTED** [review]
Just a language tweak. Do we have a release date in mind?

**Review by apburnes — APPROVED** [review]

**Comment by sknep** [comment]
I'm a little surprised to see this launch already -- lets meet Monday about the launch and announcement plan!

**Comment by svenaas** [comment]
This could be reverted pending that discussion.

**Pull request merged**